### DISCUSSION TITLE
PicView 屏蔽右上角那仨！！！我误触导致直接关闭App四五次了

### DISCUSSION THREAD

**Discussion by TPAXcc** [ywmoyue/biliuwp-lite#272] [General]
![图片](https://github.com/ywmoyue/biliuwp-lite/assets/91980905/0badcc56-7eea-4f87-94d6-3efe7e239ffc)

**Comment by ywmoyue** [comment]
能做是能做，但我感觉作为一个uwp应用，右上角的按钮没啥问题 ○|￣|_

**Comment by guaijk** [comment]
我看图片的时候也关了好几次

**Comment by TPAXcc** [comment]
> 能做是能做，但我感觉作为一个uwp应用，右上角的按钮没啥问题 ○|￣|_

但是，看图片太容易误触了啊！改成关闭 PicView 也行啊！

**Comment by ywmoyue** [comment]
看了下这块怎么搞感觉太麻烦了不想做，一定要点击右上角关闭图片的话建议启用新窗口打开图片
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/5f263d7b-1503-4558-8163-7f120c34afea)

此issue移动至Discussion
